content/app-dev/create-actor/update.md

```diff
@@ -5,6 +5,29 @@ weight: 5
 draft: false
 ---
 
+### Setting up Hot Reloading
+
+This section will introduce you to the process wasmCloud developers use to rapidly iterate on their actors. Introduced in wasmCloud `v0.50.4`, the Hot Reloading feature allows the dashboard to watch your signed actor file and automatically replace all instances of that actor upon a file change. This happens quickly (blink and you might miss it!) and tightly constrains the developer feedback loop so you can spend more time developing and less time waiting.
+
+Let's start by scaling down our `hello` actor if it's still running from a previous step. Look for the yellow scale button <img src="../scalebutton.png" alt="scalebutton" width="30px"/> in the 'Actions' column, and set the replicas to `0` before hitting Submit.
+
+![scaledown](../scaledown.png)
+
+Now, let's start the actor again using the "Start Actor" dropdown, but this time let's use the _From File (Hot Reload)_ option. You'll be required to supply the absolute path [^1] to your actor file, and you can choose a number of replicas to start as well. Luckily, we have an easy way for you to get the absolute path of your actor. Simply navigate to your actor project directory and run:
```

Comment:
does `[^1]` mean something in Hugo? just double-checking this isn't a typo

Reply:
It's a Markdown citation, it puts a hyperlinked superscript `1` there which links to the bottom of the page where I put a little more detail into why we require an absolute path. I wanted to elaborate as I'm pretty sure people would wonder why they can't just select the actor from file like they did in the previous step